lib/scala/logging-config/src/main/java/org/enso/logger/config/LogToFile.java

```diff
@@ -0,0 +1,44 @@
+package org.enso.logger.config;
+
+import com.typesafe.config.Config;
+import org.slf4j.event.Level;
+
+/**
+ * If `enabled`, all logs up to `logLevel` level will be persisted in the log file in parallel to the default appender.
+ */
+public record LogToFile(boolean enabled, Level logLevel) {
```

Comment:
Isn't there a log `Level.OFF`? It could be used to control disabling of the file logging via a single property.

Reply:
Yes, good point.

Reply:
Sadly, no `Level.OFF` in slf4j-api.